Jbarget/oxymore
# creates conscious shopping page

### What changes have you made?
- I've added in the Conscious Shopping content i.e. scrollable text and images 
- I've created a separate component called `ConsciousShoppingPreview` 
- I've added a link on the `shell` project icon so that users can return to the project page

### Which issue(s) does this PR fix?

Fixes #9 

### Screenshots (if there are design changes)

<img width="1438" alt="Screenshot 2020-10-22 at 11 17 51" src="https://user-images.githubusercontent.com/53219789/96851470-558e5180-1458-11eb-80e0-6ada23ddd9ba.png">
<img width="1438" alt="Screenshot 2020-10-22 at 11 18 28" src="https://user-images.githubusercontent.com/53219789/96851568-75257a00-1458-11eb-9b95-15c26bd3b64e.png">

### How to test
- Click on the big image in the middle and check that both images change as well as the two paragraphs
- Make sure that the translation functionality still works
- Comment out lines 158 to 186 and comment in line 187 to check the `ConsciousShoppingPreview`
- Click on the shell icon in the top left and make sure it takes you back to the projects page from both the Project page and the Preview page

------

While building out this first project page I've noticed an issue with using global background styles. At the moment we're using the marble-esque background globally. The Conscious Shopping page requires a black background but in the future there'll be other backgrounds used on the site too. I tried to tackle this on #183 but as its turned out to be a blocker on Conscious Shopping I'm asking the question here.

What we want is this:

<img width="1438" alt="95775098-06ab2400-0cc2-11eb-99b9-252f943182dc" src="https://user-images.githubusercontent.com/53219789/96852227-2a583200-1459-11eb-8b6d-90f816bdce46.png">
<img width="223" alt="95775156-293d3d00-0cc2-11eb-8f58-472659e3e458" src="https://user-images.githubusercontent.com/53219789/96852273-39d77b00-1459-11eb-87fc-b30ce554b675.png">
<img width="1440" alt="96515693-0f719c00-1266-11eb-94d6-8488e6e8d7e6" src="https://user-images.githubusercontent.com/53219789/96852278-3b08a800-1459-11eb-8be6-3b099c287146.png">

My solution: 

I looked at the React tree from the `App.js` and decided to replace the `AppContent` styled component with an actual component in which we could give it some logic. 

1. I created an array of objects containing values for page url and background 
2. I created a `GetBackground` function that returns the background depending on which page the user is on 
3. ... unsure. I _think_ I would need to pass the result of the `GetBackground` function as a prop to the `AppContent` component

I realise this might be a really long-winded solution if it makes no sense then happy to think again
